portable/GCC/ARM7_AT91SAM7S/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio input filter to be enabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsOuputSet */
+/** \brief Test if PIO Output is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsOutputSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                        unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetOutputStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_InputFilterEnable */
+/** \brief Input Filter Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_InputFilterEnable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                               unsigned int flag ) /* \arg  pio input filter to be enabled */
+    {
         pPio->PIO_IFER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_InputFilterDisable
-//* \brief Input Filter Disable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_InputFilterDisable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio input filter to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_InputFilterDisable */
+/** \brief Input Filter Disable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_InputFilterDisable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                                unsigned int flag ) /* \arg  pio input filter to be disabled */
+    {
         pPio->PIO_IFDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetInputFilterStatus
-//* \brief Return PIO Input Filter Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetInputFilterStatus( // \return PIO Input Filter Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetInputFilterStatus */
+/** \brief Return PIO Input Filter Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetInputFilterStatus( /* \return PIO Input Filter Status */
+        AT91PS_PIO pPio )                                 /* \arg  pointer to a PIO controller */
```

Comment:
Does this need to be on multiple lines?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)